### ISSUE TITLE
directory structure incorrect

### ISSUE THREAD

**Issue by klauswyser** [goord/ece2cmor3#29]
The directory structure should follow the template  on p15 in https://docs.google.com/document/d/1h0r8RZr_f3-8egBMMh7aqLwy3snpD6_MrDz1q8n5XUk/edit
```
Directory structure = <mip_era>/
<activity_id>/
<institution_id>/
<source_id>/
<experiment_id>/
<member_id>/
<table_id>/
<variable_id>/
<grid_label>/
<version>
```
where `mip_era=CMIP6` and `activity_id=HighResMIP` in the case of PRIMAVERA.

**Comment by goord** [comment]
Is the default template by CMOR correct? Then it suffices to remove the entire entry from the json file

**Comment by klauswyser** [comment]
Where can I find the CMOR template?

**Comment by klauswyser** [comment]
I tried removing 
```
"output_path_template":         "<activity_id><institution_id><source_id><experiment_id><variant_label><table><variable_id><grid_label><version>",
```
from the conf file, but then I get files with name e.g.
```
cmor/CMIP6/institute_idEC-Earth-3-HR/CMIP/historical/r1i1p1f1/Amon/tas/gr/v20170504/tas_Amon_EC-Earth-3-HR_historical_r1i1p1f1_gr_199001-199001.nc
```
I replaced <institution_id> with <institute_id> in the conf file, but then nothing works any longer.

**Comment by klauswyser** [comment]
The CMIP6_CV requires that the attribute is called <institution_id>, where does the <institute_id> come from? I did a grep on all python files in ece2cmor3 and didn't get a single hit, does it come from the CMOR3 library? In that case we maybe should ask the developers to make CMOR3 compliant with the CV requirements.

**Comment by klauswyser** [comment]
I have openend an issue https://github.com/PCMDI/cmip6-cmor-tables/issues/62

Let's see what happens.

**Comment by klauswyser** [comment]
Solved with CMOR 3.2.3
